frontend/pages/[resourceType]/[id].tsx

```diff
@@ -0,0 +1,74 @@
+import { Prism } from '@mantine/prism';
+import { Divider, Group, ScrollArea, Stack, Tabs, Text } from '@mantine/core';
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import BackButton from '../../components/BackButton';
+
+/**
+ * Component which displays the JSON/ELM/CQL/narrative content of an individual resource using
+ * Mantine tabs
+ * @returns JSON/ELM/CQL/narrative content of the individual resource in a Prism component
+ */
+export default function ResourceIDPage({ jsonData }: { jsonData: fhir4.Measure | fhir4.Library }) {
+  const router = useRouter();
+  const { resourceType, id } = router.query;
+  return (
+    <div>
+      <Stack spacing="xs">
+        <div
+          style={{
+            float: 'left'
+          }}
+        >
+          <Group>
+            <BackButton />
+            <Text size="xl" weight={700} color="gray">
+              {resourceType}/{id}
+            </Text>
+          </Group>
+        </div>
+        <Divider my="sm" style={{ paddingBottom: '5px' }} />
```

Comment:
Another nitpick but it's recommended to have padding and other spacing be consistent and scalable, which usually means multiples of 2 or 4. I usually go off of [TailwindCSS' padding system](https://tailwindcss.com/docs/padding) even though we're not using that library here (sadly). Note how a lot of their classes go from 2px, 4px, 6px, 8px, 10, 12, etc.

I'd either make this 4 or 6 instead